pkg/flag/misconf_flags.go

```diff
@@ -18,7 +20,13 @@ var (
 	ResetPolicyBundleFlag = Flag[bool]{
 		Name:       "reset-policy-bundle",
 		ConfigName: "misconfiguration.reset-policy-bundle",
-		Usage:      "remove policy bundle",
+		Usage:      "remove policy bundle, please use --reset-checks-bundle instead",
+		Deprecated: true,
+	}
+	ResetChecksBundleFlag = Flag[bool]{
+		Name:       "reset-checks-bundle",
+		ConfigName: "misconfiguration.reset-checks-bundle",
+		Usage:      "remove checks bundle",
 	}
```

Comment:
Is it possible to remove the old flag and add it as an alias to the new flag? Then you don't have to worry about handling two flags
```suggestion
	ResetChecksBundleFlag = Flag[bool]{
		Name:       "reset-checks-bundle",
		ConfigName: "misconfiguration.reset-checks-bundle",
		Usage:      "remove checks bundle",
		Aliases: []Alias{
			{
				Name:       "reset-policy-bundle",
				ConfigName: "misconfiguration.reset-policy-bundle",
				Deprecated: true,
			},
		},
	}
```

Reply:
Nice! Yeah that's much simpler to implement.